lib/pdk/util/git.rb

```diff
@@ -39,7 +39,8 @@ def self.git_with_env(env, *args)
       end
 
       def self.repo?(maybe_repo)
-        return bare_repo?(maybe_repo) if File.directory?(maybe_repo)
+        # The behavior of file://... paths remote appears to be on purpose.
```

Comment:
What behaviour in particular?

Reply:
IIRC The old behavior was that passing `--template-url /path/to/templates` did not cause a clone into the tempdir and used the directory directly, but `--template-url file:///path/to/templates` did. So the absolute paths were not considered "remote."